docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,337 @@
+Stable Patch Format
+===================
+
+Destination
+-----------
+
+Stable patches must be sent to: kernel-team@lists.ubuntu.com
+
+.. important::
+
+   please subscribe `here <https://lists.ubuntu.com/mailman/listinfo/kernel-team>`__
+   to the mailing list before sending your first patch. Messages from
+   non-subscribers will be sent to a queue waiting for an admin approval. You
+   can also manage your subscription and access the mailing list archive on
+   https://lists.ubuntu.com/mailman/listinfo/kernel-team.
+
+Subject Line
+------------
+
+#. Every patch submitted to a stable kernel **must** have have its subject line
+   starting with "[SRU]" followed by the release name against which the
+   patch is targeted. That release name **must** be enclosed in "[ ]" brackets
+   and **can** be abbreviated using the first letter of the release name (e.g.
+   "B" for "Bionic"). If a patch is to be applied to multiple releases a
+   list of release names can be provided.
+
+   Examples::
+
+     [SRU][Focal][PULL] Focal update: v5.4.56 upstream stable release
+     [SRU][B][F][PATCH 1/1] KVM: fix overflow of zero page refcount with ksm running
+
+   .. note::
+
+      * If the patch has to be applied to multiple releases and the subject
+        becomes to hard to follows, you can feel free to drop some of the
+        brackets. For example::
+
+         [SRU][B][F][J][PATCH 0/1] UBUNTU: [Config]: Add support for modifying LDT
+
+        can easily become::
+
+         [SRU][B,F,J][PATCH 0/1] UBUNTU: [Config]: Add support for modifying LDT
+
+        or::
+
+         [SRU][B/F/J][PATCH 0/1] UBUNTU: [Config]: Add support for modifying LDT
+
+      * If the patch has to be applied to a specific derivative for multiple
+        release this is the recommended format::
+
+         [SRU][B:linux-kvm,F:linux-kvm] [PATCH 0/1] UBUNTU: [Config] kvm: Add support for modifying LDT
+
+#. If the patch requested doesn't come from upstream it must contain one of
+   the following on the Subject line after the release name string
+   described previously:
+
+   .. list-table:: Example table
```

Comment:
I think this caption needs to be removed or be updated for this doc.

Reply:
Dropped. Thanks!